src/sections/dashboard/sidebar/VerticalSidebar.js

```diff
@@ -36,14 +36,20 @@ export default function VerticalSidebar() {
 
   const [isOpen, setIsOpen] = useState(true);
 
-  const otherItems = useMemo(
-    () => displayedType.children.filter((item) => item.label !== 'Setting'),
-    [displayedType],
-  );
-  const settingsItem = useMemo(
-    () => displayedType.children.find((item) => item.label === 'Setting'),
-    [displayedType],
-  );
+  const [otherItems, settingsItem] = useMemo(() => {
+    let foundSettingsItem;
+    if (displayedType && displayedType.children) {
+      const filteredItems = displayedType.children.filter((item) => {
+        if (item.label === 'Settings') {
+          foundSettingsItem = item;
+          return false;
+        }
+        return true;
```

Comment:
same [this](https://github.com/duchieupham/vietqr_web_2.0/pull/73/files#r1784619808)